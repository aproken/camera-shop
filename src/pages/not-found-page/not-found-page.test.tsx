import { render } from '@testing-library/react';
import { HelmetProvider } from 'react-helmet-async';
import { createMemoryHistory } from 'history';
import NotFoundPage from './not-found-page';
import HistoryRouter from '../../components/history-route/history-route';

describe('Component: NotFoundScreen', () => {
  it('should render correctly', () => {
    const history = createMemoryHistory();

    const {container} = render(
      <HistoryRouter history={history}>
        <HelmetProvider>
          <NotFoundPage />
        </HelmetProvider>
      </HistoryRouter>,
    );

    expect(container).toMatchSnapshot();
  });
});
