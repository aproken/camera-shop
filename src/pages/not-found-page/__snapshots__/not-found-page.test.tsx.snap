// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: NotFoundScreen should render correctly 1`] = `
<div>
  <main
    class="not-found-container"
  >
    <div
      class="not-found-content"
    >
      <h1
        class="title not-found-h1"
      >
        404
      </h1>
      <h2
        class="title not-found-h2"
      >
        page not found
      </h2>
      <a
        class="btn btn--purple"
        href="/"
        rel="stylesheet"
      >
        Вернуться на главную
      </a>
    </div>
  </main>
</div>
`;
